pkgs/applications/networking/StevenBlack-hosts/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenvNoCC
+, fetchFromGitHub
+, python3
+, lib
+, flags ? {}
+, allowList ? ""
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "StevenBlack-hosts";
+  version = "2.5.52";
+  src = fetchFromGitHub {
+    owner = "StevenBlack";
+    repo = "hosts";
+    rev = version;
+    sha256 = "01rijzfmvwfcirz94h74bx15maqdn9m01g7q197nvzxmr1fk60yd";
+  };
+
+  inherit allowList;
+  passAsFile = [ "allowList" ];
+  buildPhase =
+    let
+      py = python3.withPackages (
+        ps: with ps;[
+          lxml
+          beautifulsoup4
+        ]
+      );
+    in
+      ''
+        ln -s $allowListPath whitelist
+        ${py}/bin/python updateHostsFile.py \
+          ${lib.cli.toGNUCommandLineShell
+        {} (
+        flags // {
+          auto = true;
+          noupdate = true;
+          out = ".";
+        }
+      )
+      }
+      '';
+
+  installPhase = ''
+    odir=$out/share/StevenBlack-hosts
+    mkdir -p $odir
+    cp {.,$odir}/hosts
```

Comment:
```suggestion
    cp . $odir/hosts
```

Reply:
This would copy everything from `.`, including many build files, in `$odir` and we only want `hosts`.

Reply:
```suggestion
    cp ./hosts $odir/hosts
```

like this?